tesseract_core/sdk/cli.py

```diff
@@ -885,6 +885,14 @@ def run_container(
             show_default=False,
         ),
     ] = None,
+    network: Annotated[
+        str | None,
+        typer.Option(
+            "--network",
+            help="Network to use for the Tesseract container.",
```

Comment:
Can we add a little more info here? How would a user supply this or find more information?